easybuild/easyconfigs/w/WebKitGTK+/WebKitGTK+-2.41.4-foss-2023a.eb

```diff
@@ -0,0 +1,89 @@
+# Updated from WebKitGTK+-2.27.4-GCC-8.3.0.eb with more modules added
+# Author: J. Sassmannshausen (Imperial College London/UK)
+# Update: THEMBL
+
+easyblock = 'CMakeMake'
+
+name = 'WebKitGTK+'
+version = '2.41.4'
+
+homepage = 'https://webkitgtk.org/'
+description = """WebKitGTK+ is a full-featured port of the WebKit
+rendering engine, suitable for projects requiring any kind of web
+integration, from hybrid HTML/CSS applications to full-fledged web
+browsers. It offers WebKit's full functionality and is useful in a wide
+range of systems from desktop computers to embedded systems like phones,
+tablets, and televisions."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://webkitgtk.org/releases']
+sources = ['webkitgtk-%(version)s.tar.xz']
+checksums = ['ad792d4acd4e8536cc0a652fd0303bfef93a7f8718d2b3d72db83e25f52777af']
+
+osdependencies = [
+    # Would be better to use the system package due to security reasons
+    # But we need at least libgcrypt 1.7.0 and Ubuntu 16.04 and RH 7.5
+    # are both using a too old version.
+    # Update: Libcrypt 1.9.3 from EasyBuild added (J.S)
+    # ('libgcrypt-dev', 'libgcrypt20-dev', 'libgcrypt-devel'),
+    # libsecret is not actually needed since USE_LIBSECRET option is disabled
+    # ('libsecret-1-dev', 'libsecret-devel'),
+]
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Python', '3.11.3'),
+    ('Perl', '5.36.1', '-minimal'),
```

Comment:
we don't use `-minimal` Perl easyconfigs in 2023a anymore, should use the normal Perl here instead